resources/template_mta.yaml

```
@@ -1,16 +1,17 @@
-_schema-version: "2.0.0"
+_schema-version: "3.1"
 ID: "<Id of your MTA>"
 version: <version number of your application>
 
 parameters:
-  hcp-deployer-version: "1.0.0"
+  hcp-deployer-version: "1.1.0"
 
 modules:
   - name: "<Name of your Fiori application>"
     type: html5
     path: .
     parameters:
        version: <version number of your application>-${timestamp}
+       name: "<Name of your Fiori application>"
```

Fix code climate indentation issue:
```suggestion
        name: "<Name of your Fiori application>"
```